### PR TITLE
Update versions, changelog

### PR DESCRIPTION
**Pull request by vahancouver**
[SDKS-3362](https://bugster.forgerock.org/jira/browse/SDKS-3362) [iOS] Release activities for SDK 4.6

- Update podspec versions
- Update marketing versions
- Update sdkVersion in Log.swift
- Update Changelog
- Update PingOneSignals version from 5.2.3 to **5.2.7**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## [4.6.0]
+#### Added
+- Skip Type 4 TextOutputCallback [SDKS-3226]
+- Support PingOne Protect Marketplace Nodes [SDKS-3296]
+- Make PolicyAdviceCreator public [SDKS-3349]
+- Support new ReCaptcha Enterprise node [SDKS-3324]
+- Expose realm, Success and Failure URL in GetSessionToken [SDKS-3352]
```
[the remark this answers is not in the document]
Comment (vahancouver) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## [4.6.0]
+#### Added
+- Skip Type 4 TextOutputCallback [SDKS-3226]
```
[the remark this answers is not in the document]
Comment (vahancouver) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+- Support PingOne Protect Marketplace Nodes [SDKS-3296]
+- Make PolicyAdviceCreator public [SDKS-3349]
```
[the remark this answers is not in the document]
Comment (vahancouver) [reply CHANGELOG.md]: done

**Review by rodrigoareis — APPROVED** [review]
LGTM

**Review by spetrov — APPROVED** [review]

**Pull request merged**